src/components/Dashboard/TodoList.vue

```diff
@@ -1236,6 +1254,40 @@ export default class TodoList extends Vue {
     }
   }
 
+  private async loadConsentContinuationOut (task: ApiTaskIF): Promise<void> {
+    const filing = task.task.filing
+    const header = filing.header
+    const consentContinuationOut = filing.consentContinuationOut
+
+    if (header && consentContinuationOut) {
+      const paymentStatusCode = header.paymentStatusCode
+      const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(paymentStatusCode)
+
+      // FUTURE: check for all possible data:
+      const haveData = Boolean(consentContinuationOut.comment)
+
+      const item: TodoItemIF = {
+        name: FilingTypes.CONSENT_CONTINUATION_OUT,
+        filingId: header.filingId,
+        title: FilingNames.CONSENT_CONTINUATION_OUT,
+        draftTitle: FilingNames.CONSENT_CONTINUATION_OUT,
+        status: header.status,
+        enabled: task.enabled,
+        order: task.order,
+        paymentMethod: header.paymentMethod || null,
+        paymentToken: header.paymentToken || null,
+        payErrorObj,
+        isEmptyFiling: !haveData,
+        // FUTURE: ideally, this would come from the filing:
+        warnings: this.getBusinessWarnings.map(warning => warning.message)
+      }
+      this.todoItems.push(item)
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('ERROR - invalid header or business in filing =', filing)
+    }
+  }
+
```

Comment:
Hey, I know we're in a rush to get these changes implemented, but when we have time let's refactor this so that each filing type has it's own TODO component.  I think we'll find this simplifies the code.

Reply:
Good idea!